man/gbz80.7

```diff
@@ -556,6 +556,56 @@ Flags:
 .Ss DAA
 Decimal Adjust Accumulator to get a correct BCD representation after an arithmetic instruction.
 .Pp
+This instruction behaves like the following algorithm:
```

Comment:
I think this description is well-intentioned, but a little vague. Here is my take:
```suggestion
.Dq Decimal Adjust Accumulator :
intended to be used after performing an arithmetic instruction
.Pq Sy ADD , ADC , SUB , SBC
.Em whose inputs were in BCD
to correct its result.
.Pp
The precise behaviour of this instruction is as follows:
```

This opens with the simple guideline that suffices for 99% of use cases, with an explicit mention that it can *only* be used with BCD inputs. Then we can continue with the exact algorithm.

Reply:
What's about INC and DEC?

Reply:
Those *can* work, but they don't update flags correctly (e.g. carry); so it's arguably a more advanced use. I think the reader can derive whether it's acceptable for them based on the precise description, and the flags documented with the appropriate instruction.